.github/workflows/continuous_deployment.yml

```diff
@@ -139,27 +148,26 @@ jobs:
 
     - name: Set action server tag to existing
       # If the action server image was updated, the tag should be updated to run${{ env.BUILD_NUMBER }},
-      # otherwise it should be kept as the exisitng tag 
-      if:  ${{ needs.check_changed_files.outputs.actions == 'false' }}
+      # otherwise it should be kept as the exisitng tag
+      if: ${{ needs.check_changed_files.outputs.actions == 'false' }}
       run: |
         tag=`helm get values --output json $RELEASE_NAME | jq .app.tag`
         echo $tag
         echo "ACTION_SERVER_TAG='${tag}'" >> $GITHUB_ENV
 
     - name: Update action server tag if actions changed
       # If the action server image was updated, the tag should be updated to run${{ env.BUILD_NUMBER }},
-      # otherwise it should be kept as the exisitng tag 
-      if:  ${{ needs.check_changed_files.outputs.actions == 'true' }}
+      # otherwise it should be kept as the exisitng tag
+      if: ${{ needs.check_changed_files.outputs.actions == 'true' }}
       run: |
         tag=run$GITHUB_RUN_NUMBER-$GITHUB_RUN_ID
         echo $tag
         echo "ACTION_SERVER_TAG='$tag'" >> $GITHUB_ENV
 
     - name: Deploy Rasa X chart ☸️
-      run: |
-        cd ${{ github.workspace }}/.github/deployments && 
-        helmfile repos &&
```

Comment:
This change disabled `helmfile sync` since the second command is just `sync`

Reply:
Right, right, right 😬We need `helmfile sync` here. You are a genius!!

Reply:
Do you need a review on the PR that fix it?

Reply:
Yes please!